lib/govuk_app_config/version.rb

```diff
@@ -1,3 +1,3 @@
 module GovukAppConfig
-  VERSION = "3.1.1".freeze
+  VERSION = "4.0.0.pre-1".freeze
```

Comment:
FYI using a pre-1 name caused me a bunch of pain before: https://github.com/alphagov/rubocop-govuk/commit/be6bad2266dcce320beba08e3c831b5642c86d48. I'd recommend `pre.1`.

Reply:
Thanks for the heads up.